src/components/message-list/_macro.njk

```diff
@@ -15,10 +15,16 @@
                     <dd class="ons-message-item__metadata-value ons-message-item__metadata-value--date ons-u-fs-s">
                         {{ message.dateText }}
                     </dd>
+                    <dt class="ons-message-item__metadata-term ons-message-item__metadata-term--body ons-u-vh">
+                        {% if params.ariaLabelMsg %}
+                            {{ params.ariaLabelMsg | default("Message text") }}:
+                        {% endif %}
+                        {{ params.bodyLabel | default("Message text") }}:
```

Comment:
The way this is written allows both to be set and then both to be passed into the HTML, but we want one or the other. Currently when I test it with both params both pieces of text are rendered. We want `bodyLabel` to be the only one used if both are set. Then if one of the two are set that one is used. There is also currently a bit of an odd bug where if only `ariaLabelMsg` is set what it is set to is rendered but also "Message text:" is added on the end

<img width="594" alt="Screenshot 2025-01-14 at 09 09 20" src="https://github.com/user-attachments/assets/ff676728-3eb5-49b1-975a-8a0d13b67c0f" />

Tests need to be written for all these situations to make sure we have coverage and the test that is there will need to be checked because this should make the test fail